test/smoke/src/test-runner/test-setup.ts

```diff
@@ -118,7 +118,20 @@ function prepareTestDataDirectory() {
 }
 
 function getPositronVersion(testCodePath: string): string | null {
-	const productJsonPath = join(testCodePath, 'Contents', 'Resources', 'app', 'product.json');
+	let productJsonPath;
+	switch (process.platform) {
+		case 'darwin':
+			productJsonPath = join(testCodePath, 'Contents', 'Resources', 'app', 'product.json');
+			break;
+		case 'linux':
+			productJsonPath = join(testCodePath, 'resources', 'app', 'product.json');
+			break;
+		case 'win32':
+			productJsonPath = join(testCodePath, 'resources', 'app', 'product.json');
+			break;
+		default:
+			throw new Error('Unsupported platform.');
```

Comment:
Hmmm, maybe instead of an error we return null/undefined or `UNKNOWN`? I don't think we want to stop the test runner.

Reply:
Good point.